Hunting Queries/Corelight/CorelightMultipleRemoteSMBConnectionsFromSingleIP.yaml

```diff
@@ -0,0 +1,28 @@
+id: 69736969-6ae0-48cd-a05a-743f7372ee84
+name: Corelight - Multiple Remote SMB Connections from single client
+description: |
+  'Detects when a single source is connecting to many different SMB or file shares.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated